moped-editor/src/views/projects/projectView/ProjectComponents/index.js

```diff
@@ -212,7 +213,7 @@ export default function MapView({ projectName, projectStatuses }) {
                   }
                   onSave={onSaveEditedComponent}
                   onCancel={onCancelComponentMapEdit}
-                  saveButtonDisabled={false}
+                  saveButtonDisabled={!doesDraftEditComponentHaveFeatures}
```

Comment:
The crux of it, /and/ it explains what's happening through your great name choice.